### ISSUE TITLE
long member names - argument list a line under the member line, with a given tab space.

### ISSUE THREAD

**Issue by paule32** [doxygen/doxygen#10438]
how can I put the argument list a line under the member, so the long space are not wasteing the html page ?
it would be nice, when I could adjust the tab ident.
![grafik](https://github.com/doxygen/doxygen/assets/3682603/6b0dd806-de5c-4a08-94d9-21d46c06a490)

**Comment by albert-github** [comment]
I don't think this is directly possible as the given information is organized in a table. Which part is wasting space?
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.
- Please also specify the **full** doxygen version used (`doxygen -v`).

**Comment by paule32** [comment]
[doxy.zip](https://github.com/doxygen/doxygen/files/13449716/doxy.zip)
[GetLastError.cc.gz](https://github.com/doxygen/doxygen/files/13449721/GetLastError.cc.gz)

Version is: 1.9.6

**Comment by albert-github** [comment]
In the html/namespacewindows.html I can see the `ERROR_TOO_MANY_DESCRIPTORS() [2/2]` but you didn't indicate where you think some space is wasted.

**Comment by albert-github** [comment]
Looking at the initial issue:
Nowadays types, function names arguments tend to be much longer than in the past and thus the arguments move further to the right. So now it looks like it is a waste of space in the horizontal direction. When moving the arguments underneath the type / function line there is a waste of space in the vertical direction (extra line) whilst horizontally there might be enough space.

The `inline` issue, here the word `inline` is more a directive for the compiler on how it could handle, from the C++ draft standard:
> A function declaration (9.3.4.6, 11.4.2, 11.8.4) with an inline specifier declares an inline function. The
inline specifier indicates to the implementation that inline substitution of the function body at the point of
call is to be preferred to the usual function call mechanism. An implementation is not required to perform
this inline substitution at the point of call; however, even if this inline substitution is omitted, the other rules
for inline functions specified in this subclause shall still be respected.

So it is not part of the function signature and hence it is indicated in this way (moved to the right of the page).

I think these issues are a matter of taste.
